tests/test_fips.py

```diff
@@ -350,3 +367,23 @@ def test_nss_firefox_cert(container_per_test: ContainerData) -> None:
     c.check_output(
         'certutil -R -k rsa -g 2048 -s "CN=Daniel Duesentrieb3,O=Example Corp,L=Mountain View,ST=California,C=DE" -d "${PWD}/nssdb" -o cert9.cer -f password.txt -z seedfile.dat',
     )
+
+
+@pytest.mark.parametrize(
+    "container",
+    FIPS_OPENJDK_IMAGES,
+    indirect=True,
+)
+def test_openjdk_cert(container: ContainerData) -> None:
+    c = container.connection
+
+    # Clone, compile and run the scripts to check the FIPS provider
+    c.check_output(
+        "git clone -q https://github.com/ecki/JavaCryptoTest; cd JavaCryptoTest/src/main/java/"
+    )
+    c.check_output(
+        'javac JavaCryptoTest/src/main/java/net/eckenfels/test/jce/JCEProviderInfo.java; java -cp JavaCryptoTest/src/main/java/ net.eckenfels.test.jce.JCEProviderInfo | grep "Listing all JCA Security Providers"'
+    )
```

Comment:
you could also use the more pythonic version, which will display the full output of the call if the string is not present (might be more useful for debugging)
```suggestion
    assert "Listing all JCA Security Providers" in c.check_output(
        'javac JavaCryptoTest/src/main/java/net/eckenfels/test/jce/JCEProviderInfo.java; java -cp JavaCryptoTest/src/main/java/ net.eckenfels.test.jce.JCEProviderInfo'
    )
```

Reply:
Changed